jazz_deployments-event-handler/test/test.js

```diff
@@ -1,9 +1,327 @@
+        },
+        SERVICE_CONTEXT: {
+          S: '{"service_type":"api","branch":"","runtime":"nodejs","domain":"jazztest","iam_role":"arn:aws:iam::192006145812:role/gitlab180515_lambda2_basic_execution_1","environment":"","region":"us-east-1","message":"input validation starts","created_by":"serverless@t-mobile.com"}'
+        }
+      }
+    }
+    var encoded = Buffer.from(JSON.stringify(payload)).toString('base64');
+    var sequenceNumber = "test_sequence01";
+    var encodedPayload = encoded;
+    index.checkForInterestedEvents(encodedPayload, sequenceNumber, configData).then((res) => {
+      assert.isFalse(res.interested_event);
+    })
+  })
+})
+describe("processEventRecord", () => {
+  beforeEach(() => {
+    var payload = {
+      Item: {
+        EVENT_ID: {
+          S: '084f8c38-a01b-4ac9-943e-365f5de8ebe4'
+        },
+        TIMESTAMP: {
+          S: '2018-05-16T12:12:42:821'
+        },
+        REQUEST_ID: {
+          NULL: true
+        },
+        EVENT_HANDLER: {
+          S: 'JENKINS'
+        },
+        EVENT_NAME: {
+          S: 'CREATE_DEPLOYMENT'
+        },
+        SERVICE_NAME: {
+          S: 'test-02'
+        },
+        SERVICE_ID: {
+          S: '09ed3279-c8b9-e360-2a78-4e1ed093e6a7'
+        },
+        EVENT_STATUS: {
+          S: 'STARTED'
+        },
+        EVENT_TYPE: {
+          S: 'NOT_SERVICE_DEPLOYMENT'
+        },
+        USERNAME: {
+          S: 'serverless@t-mobile.com'
+        },
+        EVENT_TIMESTAMP: {
+          S: '2018-05-16T12:12:41:083'
+        },
+        SERVICE_CONTEXT: {
+          S: '{"service_type":"api","branch":"","runtime":"nodejs","domain":"jazztest","iam_role":"arn:aws:iam::192006145812:role/gitlab180515_lambda2_basic_execution_1","environment":"","region":"us-east-1","message":"input validation starts","created_by":"serverless@t-mobile.com"}'
+        }
+      }
+    }
+    sandbox = sinon.createSandbox();
+  })
+  afterEach(()=>{
+    sandbox.restore();
+    if(reqStub){
+      reqStub.restore();
+    }
+  })
+  it.only("should call processEvent for intrested events", () => {
+    let message = "Succesfully Updated Creation Event"
+    let responseObject = {
+      statusCode: 200,
+      body: {
+        data: {
+          message: message
+        }
+      }
+    };
+    
+    var checkForInterestedEvents = sinon.stub(index,"checkForInterestedEvents").withArgs("a","b","c").resolves("somthing");
+    var processEventStub = sinon.stub(index,"processEvent")
+    var tempAuth = "Auth_token"
+    index.processEventRecord(event.Records[0], configData, tempAuth).then((obj) => {
+    })
+  })
+  it("should Return success message when called with valid paramenters", () => {
+    let message = "Succesfully Updated Creation Event"
+    let responseObject = {
+      statusCode: 200,
+      body: {
+        data: {
+          message: message
+        }
+      }
+    };
+    reqStub = sinon.stub(request, "Request", (obj) => {
+      console.log("stub is called ")
```

Comment:
replace this with sinon.assert.calledOnce.
add this to all tests wherever you are calling the stub.